@jrohel: thanks for the suggestions. I added a commit with the suggested change.

EDIT (cf. [log](https://download.copr.fedorainfracloud.org/results/packit/rpm-software-management-dnf5-2725/fedora-rawhide-x86_64/10476462-dnf5/builder-live.log.gz)):
```
/builddir/build/BUILD/dnf5-5.4.3.0-build/dnf5-5.4.3.0/dnf5/commands/search/search_processor.cpp: In member function ‘libdnf5::cli::output::SearchResults dnf5::SearchProcessor::get_results()’:
/builddir/build/BUILD/dnf5-5.4.3.0-build/dnf5-5.4.3.0/dnf5/commands/search/search_processor.cpp:172:27: error: cannot bind non-const lvalue reference of type ‘libdnf5::rpm::Package&’ to an rvalue of type ‘libdnf5::rpm::Package’
  172 |     for (auto & package : all_matches) {
      |                           ^~~~~~~~~~~
```

--- a/dnf5/commands/search/search_processor.cpp
+++ b/dnf5/commands/search/search_processor.cpp
@@ -177,7 +177,7 @@ SearchResults SearchProcessor::get_results() {
     // In the end just push everything into the final result structure.
     SearchResults results;
     results.group_results.reserve(priority_matches.size());
-    for (auto const & [priority, packages] : priority_matches) {
+    for (auto & [priority, packages] : priority_matches) {
         results.group_results.push_back(
             {.matched_keys = get_matched_keys(priority), .matched_packages = std::move(packages)});
     }

